packages/@n8n/api-types/src/api-keys.ts

```diff
@@ -4,6 +4,8 @@ export type ApiKey = {
 	apiKey: string;
 	createdAt: string;
 	updatedAt: string;
+	/** Expiration date in unix timestamp. Null if API key never expire */
+	expiresAt: number | null;
```

Comment:
For documentaiton, we could define

```ts
/** Unix timestamp. Seconds since epoch */
export type UnixTimestamp = number
```

and use that everywhere to document what kind of type we are using